bundles/org.openhab.binding.electroluxappliances/src/main/java/org/openhab/binding/electroluxappliances/internal/dto/AirPurifierStateDTO.java

```diff
@@ -0,0 +1,346 @@
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+@NonNullByDefault
+public class AirPurifierStateDTO extends ApplianceStateDTO {
+
+    @SerializedName("properties")
+    private Properties properties = new Properties();
+
+    public Properties getProperties() {
+        return properties;
+    }
+
+    // Inner class for Properties
+    public static class Properties {
+        @SerializedName("reported")
+        private Reported reported = new Reported();
+
+        public Reported getReported() {
+            return reported;
+        }
+    }
+
+    // Inner class for Reported properties
+    public static class Reported {
+        @SerializedName("FrmVer_NIU")
+        private String frmVerNIU = "";
+
+        @SerializedName("Workmode")
+        private String workmode = "";
+
+        @SerializedName("FilterRFID")
+        private String filterRFID = "";
+
+        @SerializedName("FilterLife")
+        private int filterLife;
+
+        @SerializedName("Fanspeed")
+        private int fanspeed;
+
+        @SerializedName("UILight")
+        private boolean uiLight;
+
+        @SerializedName("SafetyLock")
+        private boolean safetyLock;
+
+        @SerializedName("Ionizer")
+        private boolean ionizer;
+
+        @SerializedName("Sleep")
+        private boolean sleep;
+
+        @SerializedName("Scheduler")
+        private boolean scheduler;
+
+        @SerializedName("FilterType")
+        private int filterType;
+
+        @SerializedName("DspIcoPM2_5")
+        private boolean dspIcoPM25;
+
+        @SerializedName("DspIcoPM1")
+        private boolean dspIcoPM1;
+
+        @SerializedName("DspIcoPM10")
+        private boolean dspIcoPM10;
+
+        @SerializedName("DspIcoTVOC")
+        private boolean dspIcoTVOC;
+
+        @SerializedName("ErrPM2_5")
+        private boolean errPM25;
+
+        @SerializedName("ErrTVOC")
+        private boolean errTVOC;
+
+        @SerializedName("ErrTempHumidity")
+        private boolean errTempHumidity;
+
+        @SerializedName("ErrFanMtr")
+        private boolean errFanMtr;
+
+        @SerializedName("ErrCommSensorDisplayBrd")
+        private boolean errCommSensorDisplayBrd;
+
+        @SerializedName("DoorOpen")
+        private boolean doorOpen;
+
+        @SerializedName("ErrRFID")
+        private boolean errRFID;
+
+        @SerializedName("SignalStrength")
+        private String signalStrength = "";
+
+        @SerializedName("logE")
+        private int logE;
+
+        @SerializedName("logW")
+        private int logW;
+
+        @SerializedName("InterfaceVer")
+        private int interfaceVer;
+
+        @SerializedName("VmNo_NIU")
+        private String vmNoNIU = "";
+
+        @SerializedName("TVOCBrand")
+        private String tvocBrand = "";
+
+        private Capabilities capabilities = new Capabilities();
+
+        private Tasks tasks = new Tasks();
+
+        @SerializedName("$version")
```

Comment:
This $ is no typo? not seen this before, is there a special use?

Reply:
<img width="472" alt="image" src="https://github.com/user-attachments/assets/485b4608-62d7-44d4-8e33-fced45bb2f96">

Not sure why they use $, maybe some system-reserved or metadata field?